adminer/drivers/pgsql.inc.php

```diff
@@ -721,7 +722,11 @@ function alter_indexes($table, $alter) {
 			} elseif ($val[2] == "DROP") {
 				$drop[] = idf_escape($val[1]);
 			} else {
-				$queries[] = "CREATE INDEX " . idf_escape($val[1] != "" ? $val[1] : uniqid($table . "_")) . " ON " . table($table) . ($val[3] ? " USING $val[3]" : "") . " (" . implode(", ", $val[2]) . ")";
+				$index_condition = "";
+				if ($val[4]) {
+					$index_condition = " WHERE ".$val[4];
+				}
+				$queries[] = "CREATE INDEX " . idf_escape($val[1] != "" ? $val[1] : uniqid($table . "_")) . " ON " . table($table) . ($val[3] ? " USING $val[3]" : "") . " (" . implode(", ", $val[2]) . ")".$index_condition;
```

Comment:
Wrap this to more lines using logical blocks. Then you can inline `$index_condition` here.